pkg/controller/imagebuild/component/builddispatcher.go

```diff
@@ -231,7 +235,7 @@ func (c *BuildDispatcherComponent) Reconcile(coreCtx *core.Context) (ctrl.Result
 
 	// best effort phase change regardless if the original context is "done"
 	coreCtx.Context = context.Background()
-	imageSize, err := bk.Build(buildCtx, buildOpts)
+	imageName, err := bk.Build(buildCtx, buildOpts)
```

Comment:
So you can get image size from the build itself but need to fetch the image post-build in order to get additional information like labels?